content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,675 @@
+---
+title: Updating Configuration via a ConfigMap
+content_type: tutorial
+weight: 20
+---
+
+<!-- overview -->
+This page provides a step-by-step example of updating configuration within a Pod via a ConfigMap
+and builds upon the [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) task.  
+At the end of this tutorial, you will understand how to change the configuration for a running application.   
+This tutorial uses the `alpine` and `nginx` images as examples.
+
+## {{% heading "prerequisites" %}}
+{{< include "task-tutorial-prereqs.md" >}}
+
+You need to have the [curl](https://curl.se/) command-line tool for making HTTP requests from
+the terminal or command prompt. If you do not have `curl` available, you can install it. Check the
+documentation for your local operating system.
+
+## {{% heading "objectives" %}}
+* Update configuration via a ConfigMap mounted as a Volume
+* Update environment variables of a Pod via a ConfigMap
+* Update configuration via a ConfigMap in a multi-container Pod
+* Update configuration via a ConfigMap in a Pod possessing a Sidecar Container
+
+<!-- lessoncontent -->
+
+## Update configuration via a ConfigMap mounted as a Volume {#rollout-configmap-volume}
+
+Use the `kubectl create configmap` command to create a ConfigMap from [literal values](/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-literal-values):
+
+```shell
+kubectl create configmap sport --from-literal=sport=football
+```
+Below is an example of a Deployment manifest with the ConfigMap `sport` mounted as a
+{{< glossary_tooltip text="volume" term_id="volume" >}} into the Pod's only container.
+{{% code_sample file="deployments/deployment-with-configmap-as-volume.yaml" %}}
+
+Create the Deployment:
+```shell
+kubectl apply -f https://k8s.io/examples/deployments/deployment-with-configmap-as-volume.yaml
+```
+
+Check the pods for this Deployment to ensure they are ready (matching by
+{{< glossary_tooltip text="selector" term_id="selector" >}}):
+```shell
+kubectl get pods --selector=app.kubernetes.io/name=configmap-volume
+```
+
+You should see an output similar to:
+```
+NAME                                READY   STATUS    RESTARTS   AGE
+configmap-volume-6b976dfdcf-qxvbm   1/1     Running   0          72s
+configmap-volume-6b976dfdcf-skpvm   1/1     Running   0          72s
+configmap-volume-6b976dfdcf-tbc6r   1/1     Running   0          72s
+```
+
+On each node where one of these Pods is running, the kubelet fetches the data for that ConfigMap and translates it to files in a local volume. The kubelet then mounts that volume into the container, as specified in the Pod template. The code running in that container loads the information from the file and uses it to print a report to stdout. You can check this report by viewing the logs for one of the Pods in that Deployment:
```

Comment:
Please do wrap long lines like this.

Reply:
Ack. This will be handled in the next commit.